docfx_yaml/extension.py

```diff
@@ -1260,6 +1261,117 @@ def find_markdown_pages(app, outdir):
             })
 
 
+# Finds and replaces occurrences which should be a cross reference in the given
+# content, except for the current name.
+def convert_cross_references(content, current_name, sorted_map):
+    words = content.split(" ")
+    new_words = []
+    # Using counter to check if the entry is already a cross reference.
+    for index, word in enumerate(words):
+        cross_reference = ""
+        for keyword in sorted_map:
+            if keyword != current_name and keyword not in current_name and keyword in word:
```

Comment:
Why is it `keyword in word` instead of equals?

Reply:
Because we break by space, we could encounter situations like `` `google.cloud.bigquery_storage_v1.types` `` and other similar forms where the cross reference exists within special characters or as part of some form of one line syntax.